.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "cecil"]
 	path = external/cecil
-	url = git://github.com/mono/cecil.git
+	url = https://github.com/mono/cecil.git
```

Comment:
@akoeplinger should this be updated or we just rely on GitHub's auto-redirect?

(just asking, have no preference here)

Reply:
yeah, good idea to be consistent: https://github.com/dotnet/linker/pull/3132